src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -693,4 +705,9 @@ private void workAroundHttpClientCloneBug() {
             alignment.setCigar(new ArrayList<>(alignment.getCigar()));
         }
     }
+
+    @Override
+    public String toString() {
+        return uuid.toString() + ":" + genomicsRead.toString();
```

Comment:
This will likely change once https://github.com/broadinstitute/hellbender/issues/624 is implemented -- is anything in this branch sensitive to changes in the string representation?